### PR TITLE
feat(portals-admin-ids-admin): Adding client secrets

### PR DESCRIPTION
**Pull request by saevarma**
## What

* [Reveal and copy secrets in UI](https://www.notion.so/Update-client-secret-UI-to-allow-revealing-and-copying-the-client-secret-82c0a08f4164492cb49f8f704c40d921?pvs=4)
* [Automatically create secrets on new client](https://www.notion.so/Automatically-create-a-secret-when-creating-a-new-client-41254ab7578d4e489badcee5884d642d?pvs=4)
* [GQL field resolver to get secrets](https://www.notion.so/Add-GraphQL-field-resolver-to-client-model-to-get-secrets-ddf5f5e1228f4c469166c721ba0915fc?pvs=4)
* [Add openid and profile scopes to new client](https://www.notion.so/Add-openid-and-profile-scopes-to-client-on-creation-db8817f21da14a85be355b48384e9647?pvs=4)

## Why

To access secrets for new clients. 
Added `openid` and `profile` scopes automatically on client creation to enable clients for authorization.

## Screenshots / Gifs

Client with new secret
image

Client with legacy secret
image

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/portals/admin/ids-admin/src/components/Client/AdvancedSettings.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ interface AdvancedSettingsProps {
   allowOfflineAccess: boolean
   requireConsent: boolean
   supportTokenExchange: boolean
-  slidingRefreshTokenLifetime: number
+  accessTokenLifetime: number
```
Comment (snaerseljan) [inline libs/portals/admin/ids-admin/src/components/Client/AdvancedSettings.tsx:22]: ~nit: I know this is not your doing but couldn't this be picked from generated code, i.e. all fields?

File: libs/portals/admin/ids-admin/src/components/Client/BasicInfo.tsx [hunk 2]
```diff
@@ -1,48 +1,53 @@
...
+
 import { m } from '../../lib/messages'
 import ContentCard from '../../shared/components/ContentCard'
-import { toast } from '@island.is/island-ui/core'
+import { AuthAdminClientSecret } from './Client.loader'
 
 interface BasicInfoProps {
   clientId: string
-  clientSecret?: string
+  clientSecrets?: AuthAdminClientSecret | null
```
Comment (snaerseljan) [inline libs/portals/admin/ids-admin/src/components/Client/BasicInfo.tsx:18]: You need to update the type here from the generated code to the following type
`GetClientQuery['authAdminClient']['environments'][0]['secrets']`

**Review by snaerseljan — APPROVED** [review]
Amazing work @saevarma 💯 

Really nice code and structure. Just need to fix one type and you should be good.

File: libs/portals/admin/ids-admin/src/components/Client/BasicInfo.tsx (hunk 2 above)
Comment (saevarma) [reply libs/portals/admin/ids-admin/src/components/Client/BasicInfo.tsx:18]: Already was 😉

File: libs/portals/admin/ids-admin/src/components/Client/AdvancedSettings.tsx (hunk 1 above)
Comment (saevarma) [reply libs/portals/admin/ids-admin/src/components/Client/AdvancedSettings.tsx]: Done

**Review by snaerseljan — APPROVED** [review]

**Pull request merged**